ckan/templates/organization/edit_base.html

```diff
@@ -22,6 +22,11 @@
     {{ h.build_nav_icon(group_type + '.edit', _('Edit'), id=group_dict.name, icon='pencil') }}
     {{ h.build_nav_icon(group_type + '.bulk_process', h.humanize_entity_type('package', dataset_type, 'content tab') or _('Datasets'), id=group_dict.name, icon='sitemap') }}
     {{ h.build_nav_icon(group_type + '.manage_members', _('Members'), id=group_dict.name, icon='users') }}
+    {# (canada fork only): background search index rebuilding #}
+    {#  TODO: upstream contrib!! #}
+    {% if h.check_access('sysadmin') %}
```

Comment:
you can reference the actual action here in case someone overrides the auth function